test/trial_runner_test.py

```diff
@@ -0,0 +1,200 @@
+from __future__ import absolute_import
```

Comment:
Looks like we aren't running this test anywhere. Should we run it in Travis (that would require pip installing gym in Travis)? or Jenkins?

We may want to go with Jenkins since we'll be able to run longer jobs (it currently lacks Python 3 testing, but we can build a docker container with Python 3 and potentially do both in Jenkins).

Reply:
This test only takes 45s since we use mocked agents (and it shouldn't depend on gym).

How do I enable it in travis?

Reply:
Basically need to add it somwhere around here https://github.com/ray-project/ray/blob/3764f2f2e1447f4015d4e2bcd47700131e85bc80/.travis.yml#L122